tests/process.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+import csv
+
+def compare_string_sets(strings1, strings2):
+    set1 = set(strings1)
+    set2 = set(strings2) 
+    return (len(set1)==len(set2))&all(any(s1 in s2 for s1 in set1) for s2 in set2)
```

Comment:
This looks pretty complex, so I am not sure exactly what this does (a docstring might be helpful here), but using `&` is not Pythonic